openslides_backend/action/actions/projector/toggle.py

```diff
@@ -85,8 +87,14 @@ def get_updated_instances(self, action_data: ActionData) -> ActionData:
                         "options": instance.get("options"),
                         "meeting_id": meeting_id,
                     }
+                    if not stable:
+                        self.move_all_unstable_projections_to_history(
+                            projector_id, meeting_id
+                        )
+                        scroll_to_0 = True
```

Comment:
Just curious: Why can't you yield the projector update here?

Reply:
Ah, okay, I dont need that code, I can just use the yield here. Okay.